### PR TITLE
Json web token

### PR DESCRIPTION
**Pull request by bhargavisatpathy**

### CODE REVIEW

File: _posts/2016-08-08-jwt.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+Traditionally, a server saves the authenticated session, and session cookie gives the client access. Nowadays, communication with scaled out services has to be stateless. Also, multi-tenant service needs to be authorized. This is where JWT comes in handy.
+
+JSON Web Token (JWT) is an open standard [RFC 7519](https://tools.ietf.org/html/rfc7519) that defines a *compact* and *self-contained* way for securely transmitting (user) information between parties as a JSON object. Because of its smaller size, JWTs can be sent through an URL, POST parameter, or inside an HTTP header. Additionally, the smaller size means transmission is fast. Because the payload contains all the required information about the user, it avoids the need to query the database more than once. These two aspects makes JWT suitable for stateless communication. Additionally, JWT can be verified and trusted because it is digitally signed with public/private key pair.
```
Comment (doomspork) [inline _posts/2016-08-08-jwt.md]: @bhargavisatpathy did you write this or copy it from JWT.io?  I noticed this read _just_ like that site so I took a peek and this is almost verbatim what they have on their site.

File: _posts/2016-08-08-jwt.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+This is a stateless authentication mechanism as the user state is never saved in server memory. The server's protected routes will check for a valid JWT in the Authorization header, and if it's present, the user will be allowed to access protected resources.
+
+This allows you to fully rely on data APIs that are stateless and even make requests to downstream services. It doesn't matter which domains are serving your APIs, so Cross-Origin Resource Sharing (CORS) won't be an issue as it doesn't use cookies.
+
+The following diagram shows this process:
+
+<img style="float: left; padding-right: 20px;" src="{{ site.url }}/assets/images/jwt.png">
+
+
+### Benefits of JSON Web Tokens (JWT)
```
Comment (doomspork) [inline _posts/2016-08-08-jwt.md]: This whole section appears to be copied from Auth0: https://auth0.com/learn/json-web-tokens/

**Comment by doomspork** [comment]
I'm quite excited to see a JWT post but I think this is a bit too much copy and paste of other people's content.  Having visited JWT.io and Auth0 just a few times I knew this was their content by how it reads.  I think we should put a little more effort and change up the wording so it's not verbatim.

**Comment by bhargavisatpathy** [comment]
@doomspork Yes, they are the authority in this standard. I wrote the blog based on what I read and liked. Do you want me to put attribution to that site?

**Comment by doomspork** [comment]
@bhargavisatpathy it's one thing to read something and write a blog post on it, this is verbatim copy and paste of those sources which is I think is a bad idea.  The wording hasn't even been changed.

I'm not in charge of the blog but I certainly think copying and pasting the authoritative sources and claiming it as our content is more likely to reflect negatively then positively on us.  Plus, what's the point?  There's no new information in this since it's just those two site's content.

**Pull request closed without merging**